server/src/resolversImpl.ts

```diff
@@ -93,11 +94,30 @@ export async function discoverCluster(
   context.logger.debug(`Fetching flows for discovery...`);
   const flowsConnection = await context.database.getFlows(
     context,
+    { filterBy },
     {
-      filterBy
-    },
-    1000000,
-    true
+      processL7: true,
+      flowsFiltersExtensionCallback: ({
```

Comment:
i think you need to give a list of filters instead of specifying all the conditions in a single filter. conditions in a filter gets ANDed.

Reply:
Got it, check updates please